adapters/invibes/invibes.go

```diff
@@ -0,0 +1,327 @@
+package invibes
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/golang/glog"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"github.com/prebid/prebid-server/pbsmetrics"
+)
+
+const adapterVersion = "prebid_1.0.0"
+
+type InvibesAdRequest struct {
+	BidParamsJson string
+	Location      string
+	Lid           string
+	IsTestBid     bool
+	Kw            string
+	IsAmp         bool
+	Width         string
+	Height        string
+	GdprConsent   string
+	Gdpr          bool
+	Bvid          string
+	InvibBVLog    bool
+	VideoAdDebug  bool
+}
```

Comment:
Nitpick: Per Go naming conventions, acronyms should be spelled out in uppercase. That would be:
IsAmp -> IsAMP
GdprConsent -> GDPRConsent
Gdpr -> GDPR
PlacementIds -> PlacementIDs
ImpId -> ImpID